library/src/main/java/com/google/android/exoplayer/util/MimeTypes.java

```diff
@@ -53,6 +53,7 @@
   public static final String AUDIO_OPUS = BASE_TYPE_AUDIO + "/opus";
 
   public static final String TEXT_VTT = BASE_TYPE_TEXT + "/vtt";
+  public static final String TEXT_TX3G = BASE_TYPE_TEXT + "/tx3g";
```

Comment:
I think this should be:
public static final String APPLICATION_TX3G = BASE_TYPE_APPLICATION + "/x-quicktime-tx3g";